eva/parser/eva.lark

```diff
@@ -41,6 +41,9 @@ udf_impl: string_literal
 
 index_type: USING HNSW
 
+index_elem: ("(" uid_list ")"
```

Comment:
I think we only support single column right now.

Reply:
Yes. I had this discussion with @gaurav274. We agree to keep this in the parser but do a check in the statement binder.